db/agents.yml

```diff
@@ -317,13 +326,20 @@ agents:
       - Amazon;Echo_Dot;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422
       - Amazon;Echo_Dot_with_clock;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422
       - Amazon;Echo_Show_5;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422
-  - regex: '^English.+Radio.+IELTS.+TOEFL.+CFNetwork'
+  - regex: '^English.*Radio.+CFNetwork'
     name: English Radio IELTS TOEFL
     type: Mobile App
     os: iOS
     examples:
       - English%20Radio%20-%20IELTS%20TOEFL%20TOEIC%20Listening%20and%20Speaking/1 CFNetwork/976 Darwin/18.2.0
       - English Radio - IELTS TOEFL TOEIC Listening and Speaking/1 CFNetwork/976 Darwin/18.2.0
+      - EnglishRadio/1 CFNetwork/1498.700.2 Darwin/23.6.0
+  - regex: '^ESP32 HTTP Client'
+    name: null
+    type: Smart Home
+    os: Arduino
+    examples:
+      - ESP32 HTTP Client/1.0
```

Comment:
I'm mostly sure these are from [Yoto](https://us.yotoplay.com/) ... but they're not overriding the default UA from this [IoT framework](https://github.com/espressif/esp-idf/tree/master).

Reply:
I didn't call this an `app` ... just `type` and `os`.  Or do you think those aren't going to be accurate either?

Reply:
Making any sort of OS call for an ESP32 device is going to be tough, if the only thing we know is that its an ESP32 decice. Most run freeRTOS I think, if they have an OS at all. I don't think it would generally be accurate to call the OS Arduino, maybe ever.

Type=Smart Home is probably the closest thing we have, though likely used in this way would lead to some inaccuracies. Something like IoT, MCU or Embdedded could be better.

Reply:
Okay, `os` gone!

Left as smart-home for now, just to have something there.